diff --git a/meson.build b/meson.build
index 15fd5cd..354eeb2 100644
--- a/meson.build
+++ b/meson.build
@@ -8,7 +8,7 @@ project('gcab', 'c',
 git_version = []
 git = find_program('git', required: false)
 if git.found()
-  git_version = run_command(git, 'describe', '--abbrev=4', '--dirty', check: true).stdout().strip().split('-')
+  git_version = run_command(git, '--version', check: true).stdout().strip().split(' ')
 endif
 
 # libtool versioning
